var/spack/repos/builtin/packages/py-pyamg/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPyamg(PythonPackage):
+    """PyAMG is a library of Algebraic Multigrid (AMG) solvers with
+    a convenient Python interface."""
+
+    homepage = "https://github.com/pyamg/pyamg"
+    url      = "https://github.com/pyamg/pyamg/archive/v4.0.0.zip"
+
+    # A list of GitHub accounts to notify when the package is updated.
+    maintainers = []
```

Comment:
Are you sure you don't want to add your GitHub account to indicate you are a maintainer of the package?

Reply:
I’m not the maintainer. Those are @lukeolson and @jbschroder. Or do you mean maintainer of this Spack package file?

Reply:
I mean of the Spack package.

Reply:
OK, done.

Reply:
Great!  Thanks.